src/features/MergeNetworks/components/MergeDialog.css

```diff
@@ -0,0 +1,73 @@
+.p-dropdown-panel {
```

Comment:
I think the current convention for this codebase is to not make separate css files for components, the css is just inline in the `sx` or `style` props.  

@keiono any thoughts on this?  

In my opinion, not a big deal for now though.

Reply:
Emmm, if I use cx or style, I think it will cause some code duplications since there are some components with the same class sharing the same style.